interface/patient_file/summary/rx_left.php

```diff
@@ -18,6 +37,14 @@
 <span><?php xl('Add','e');?></span></a>
 </td>
 </tr>
+<tr>
+<td>
+<?php if($GLOBALS['rx_show_drug-drug']){ ?>
+<a href="<?php echo $GLOBALS['webroot']?>/interface/weno/drug-drug.php"  target='RxRight' class="css_button" onclick="top.restoreSession()">
+<span><?php xl('Drug-Drug', 'e');?></span></a>
+<?php } ?>
```

Comment:
indent here(with 4 spaces) and also use echo xlt:
``````
<?php if($GLOBALS['rx_show_drug-drug']){ ?>
    <a href="<?php echo $GLOBALS['webroot']?>/interface/weno/drug-drug.php"  target='RxRight' class="css_button" onclick="top.restoreSession()">
    <span><?php echo xlt('Drug-Drug');?></span></a>
<?php } ?>